<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogContentsList</class>
 <widget class="QDialog" name="DialogContentsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Folder Contents</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QCheckBox" name="chbTop10">
     <property name="text">
      <string>Top10</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTotal">
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="ClickableLabel" name="labelFolderName">
     <property name="text">
      <string>folder name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="WidgetFileTypes" name="types_">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Files Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Size</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>src/clickablelabel.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetFileTypes</class>
   <extends>QTreeWidget</extends>
   <header>src/widgetfiletypes.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
